src/custom-element.js

```diff
@@ -40,6 +40,7 @@ import {
 import * as dom from './dom';
 import {setStyle, setStyles} from './style';
 import {LayoutDelayMeter} from './layout-delay-meter';
+import {ResourceState} from './service/resource';
```

Comment:
@dvoytenko PTAL as you may not like this 😨 . This passed dep-check, but I am not sure it is a good idea to import something from `resource` to `customElement`